nightmare_firmware/servos.cpp

```diff
@@ -100,11 +102,9 @@ void servo::attach(){
   	servo_tx_enb();
   
   	servo_write(
-		SERVO_HEADER,
 		id,
 		SERVO_LOAD_OR_UNLOAD_WRITE,
-		0x01,
-		~((byte)(id+0x24))
+		0x01
   	);
```

Comment:
same here, aren't you missing something?here you get `id+0x01+4+31` (4 and 31 are from `SERVO_LOAD_OR_UNLOAD_WRITE`)